src/lib/UserEventHelper.ts

```diff
@@ -21,6 +21,23 @@ export function verifyStatus(status: any) {
   return stat;
 }
 
+export async function verifyHost(eid: string, uid: string) {
```

Comment:
This is a duplicate of `validateHost()` function in `EventHelper.ts`. Use that one instead.